configator/generate_readme.py

```diff
@@ -0,0 +1,11 @@
+"""Element of the Configator tool to enable default README.md generation"""
+
+print("Generating default README...!")
```

Comment:
Global-level print statements should be avoided -- these are executed when the containing python module is imported, and so do not run in the order you may be imagining. Instead, they should be inside the function that will actually do the generation.